packages/snippetz/playground/src/components/CodeExample.vue

```diff
@@ -1,16 +1,12 @@
 <script setup lang="ts">
 import { snippetz } from '@scalar/snippetz'
-import {
-  type ClientId,
-  type TargetId,
-  objectToString,
-} from '@scalar/snippetz/core'
+import { type TargetId, objectToString } from '@scalar/snippetz/core'
 import { getHighlighter } from 'shikiji'
 import { onMounted, ref, watch } from 'vue'
 
 const props = defineProps<{
-  target: TargetId
```

Comment:
is there a reason why we opted for string instead of a ClientID type @hanspagel ?

Reply:
Thanks for bringing this up, I’ll have to fix the types in a separate PR, though. ✌️ There’s just too much that we need to change.